srcs/incs/HttpRequestHandler.hpp

```diff
@@ -14,20 +15,55 @@ class HttpRequestHandler : public IRequestHandler {
 		bool				GetKeepAlive() const;
 
 	private:
+		HttpRequestHandler();
+		HttpRequestHandler(const HttpRequestHandler &);
+		HttpRequestHandler&	operator=(const HttpRequestHandler&);
+
 		const ServerConfig	&server_config_;
 		std::string			raw_request_;
 		std::string			raw_response_;
 		bool				keep_alive_;
 
+		void				SetKeepAlive_(const HttpRequest &request);
+		const Location		*FindLocation_(
+										const std::string &request_path) const;
+		std::string			GetReturnUrl_(const Location *location) const;
```

Comment:
To be honest, to me, all of these getters for Location would make sense in the Location class (struct ?). Consequentially, also the setters.

Reply:
I have changed it. Please, tell me what you think.